README.md

```diff
@@ -229,6 +229,7 @@ _Source:_ [What is Docker](https://www.docker.com/why-docker)
 - [Panamax](https://github.com/CenturyLinkLabs/panamax-ui) :skull: - An open-source project that makes deploying complex containerized apps as easy as Drag-and-Drop by [@CenturyLinkLabs][centurylinklabs].
 - [Rancher](https://github.com/rancher/rancher) - An open source project that provides a complete platform for operating Docker in production by [@rancher][rancher].
 - [Swarm-cronjob](https://github.com/crazy-max/swarm-cronjob) - Create jobs on a time-based schedule on Swarm by [@crazy-max](crazy-max)
+- [Pdocker](https://github.com/g31s/Pdocker) - A simple tool to manage and maintain Docker for personal projects by [@g31s](g31s)
```

Comment:
Not in alphabetical order and in the wrong section.

Please add it here: https://github.com/veggiemonk/awesome-docker/pull/734/files#diff-04c6e90faac2675aa89e2176d2eec7d8L333